frontends/web/src/components/transactions/transaction.tsx

```diff
@@ -330,6 +331,12 @@ class Transaction extends Component<Props, State> {
                                     </div>
                                 ) : null
                             }
+                            {
```

Comment:
actually that is no good. the nonce has to be `undefined | number`, since it does not exist for BTC but can be 0 in eth (first tx).

Reply:
done

Reply:
:+1: yeah, @thisconnect would be better here.
from the UI point of view though, not sure how useful it would be for me as a user. wouldn't know what to do with it. feels more like a debugging output.